tests/bootstrap.js

```diff
@@ -65,6 +71,22 @@ const waitForPg = async (url) => {
   }});
 };
 
+const isDBReady = (url = '') => {
+  return new Promise((resolve, reject) => {
+    http
```

Comment:
Going native, because 1 usage for testing and to ensure long live across node versions.